pkg/kubelet/image_manager_test.go

```diff
@@ -37,6 +37,7 @@ func newRealImageManager(policy ImageGCPolicy) (*realImageManager, *container.Fa
 	return &realImageManager{
 		runtime:      fakeRuntime,
 		policy:       policy,
+		minAge:       0,
```

Comment:
Good point. To be honest, I considered to add such a test. But if we add such a test, the gc routine need ***sleep*** for a 'minAge' period or so after images are added, which will make the test run for a longer time. So, how to implement it? set `minAge` a small value(about 5 second) so that the gc routine would not sleep for a long time, or try to find some other *effective* approach to write the unit test.